src/process_utility.c

```diff
@@ -282,9 +283,20 @@ check_altertable_add_column_for_compressed(Hypertable *ht, ColumnDef *col)
 					has_notnull = true;
 					continue;
 				case CONSTR_DEFAULT:
-					/* since default expressions might trigger a table rewrite we
-					 * only allow Const here for now */
-					if (!IsA(constraint->raw_expr, A_Const))
+					/*
+					 * Since default expressions might trigger a table rewrite we
+					 * only allow Const here for now.
```

Comment:
is this comment about `Const` valid still?

Reply:
yes